CHANGELOG.md

```diff
@@ -65,6 +65,26 @@ Replace any use of this class with the `govuk-tag--grey` class.
 
 This change was introduced in [pull request #2417: Remove deprecated `govuk-tag--inactive class`](https://github.com/alphagov/govuk-frontend/pull/2417).
 
+#### Make sure any individually imported JavaScript modules work as expected
+
+We've changed the naming of our component JavaScript modules so that individual imports are now attached to
+`window.GOVUKFrontend.[ComponentName]` instead of `window.GOVUKFrontend`.
+
+This means you can now import multiple modules without overwriting the previous one, for example:
+
+```
+//= require govuk/components/accordion/accordion.js
+//= require govuk/components/button/button.js
+
+# The above are available under window.GOVUKFrontend.Accordion and window.GOVUKFrontend.Button respectively
+```
+
+If you are importing indvidual component JavaScript, you should check any references to `window.GOVUKFrontend` in your code and update them to point to the correct component: `window.GOVUKFrontend.[ComponentName]`. You may have implemented workarounds that can now be removed.
+
+This change does not affect you if you followed our [Getting Started guide](https://frontend.design-system.service.gov.uk/get-started/#5-get-the-javascript-working) and are importing all of the GOV.UK Frontend JavaScript in one go via `all.js`, or if you installed GOV.UK Frontend using precompiled files.
```

Comment:
Wonder should this display right under the heading? Just to help users who do not need to do anything skip over this content.

```suggestion
You do not need to do anything if you have either:

- followed our [Getting Started guide](https://frontend.design-system.service.gov.uk/get-started/#5-get-the-javascript-working) and are importing all of the GOV.UK Frontend JavaScript in one go via `all.js`
- installed GOV.UK Frontend using precompiled files

```

Reply:
Good idea, I've moved this to the top and made the suggestions you mentioned